cartridges/int_alma/cartridge/templates/default/checkout/billing/paymentOptions/paymentOptionsContent.isml.example

```diff
@@ -1,8 +1,16 @@
+<isinclude template="checkout/billing/paymentOptions/almaIncludes" />
+
 <isloop items="${pdict.order.billing.payment.applicablePaymentMethods}" var="paymentOption">
     <isif condition="${paymentOption.ID === 'CREDIT_CARD'}">
         <isinclude template="checkout/billing/paymentOptions/creditCardContent" />
     </isif>
-    <isif condition="${pdict.isAlmaEnable && paymentOption.ID === 'ALMA'}">
-        <isinclude template="checkout/billing/paymentOptions/almaContent" />
+    <isif condition="${pdict.isAlmaEnable && paymentOption.ID === 'ALMA_DEFERRED'}">
+        <isinclude template="checkout/billing/paymentOptions/almaContent/almaContentDeferred" />
+    </isif>
+    <isif condition="${pdict.isAlmaEnable && paymentOption.ID === 'ALMA_PNX'}">
+        <isinclude template="checkout/billing/paymentOptions/almaContent/almaContentPnx" />
+    </isif>
+    <isif condition="${pdict.isAlmaEnable && paymentOption.ID === 'ALMA_CREDIT'}">
+        <isinclude template="checkout/billing/paymentOptions/almaContent/almaContentCredit" />
```

Comment:
It would be nice to update [this document](https://docs.getalma.eu/docs/cartridge-installation-guide#c-afficher-alma-comme-methode-de-paiement)
for this part, but maybe for the config part too, wdyt ?

Also, a little explanation on what are `ALMA_DEFERRED` `ALMA_PNX` and `ALMA_CREDIT`, and how they probably don't need all of them would be great !